func_adl_xAOD/common/statement.py

```diff
@@ -140,7 +140,7 @@ def emit(self, e):
 
 
 class set_var:
-    'Set a variable to a value'
+    'Set a value to a variable'
```

Comment:
Set a variable to a value is correct, but perhaps a better way to write this is "assign a value to a variable".